scripts/dev/host_deploy_ipks

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Host an OE IPK deploy tree on an HTTP server."""
+
+from argparse import ArgumentParser, Namespace
+from glob import glob
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+import logging
+from pathlib import Path
+import os
```

Comment:
Alphabetically should go before `from pathlib..`. Also a good idea to run https://pypi.org/project/flake8/, https://pypi.org/project/black/.

Reply:
Will fix.